pkg/controller/backupstoragelocation_controller.go

```diff
@@ -44,49 +57,62 @@ func (r *BackupStorageLocationReconciler) Reconcile(req ctrl.Request) (ctrl.Resu
 
 	log.Info("Checking for existing backup locations ready to be verified; there needs to be at least 1 backup location available")
 
-	locationList, err := velero.ListBackupStorageLocations(r.StorageLocation.Client, r.StorageLocation.Ctx, req.Namespace)
+	locationList, err := storage.ListBackupStorageLocations(r.Ctx, r.Client, req.Namespace)
 	if err != nil {
 		log.WithError(err).Error("No backup storage locations found, at least one is required")
 		return ctrl.Result{Requeue: true}, err
 	}
 
+	pluginManager := r.NewPluginManager(log)
+	defer pluginManager.CleanupClients()
+
 	var defaultFound bool
 	var unavailableErrors []string
 	var anyVerified bool
 	for i := range locationList.Items {
 		location := &locationList.Items[i]
 		log := r.Log.WithField("controller", "backupstoragelocation").WithField("backupstoragelocation", location.Name)
```

Comment:
can this reuse `log` from line 56?

Reply:
Not if we want to log the location name.

Reply:
Line 74 is instantiating a new logger in every iteration.
Can we use something like
```
bslLogger := log.WithField("backupstoragelocation", location.Name)
```
reusing `log` from line 56

Reply:
probably not instantiating a new logger.

Reply:
Yes, and `log.WithField("backupstoragelocation", location.Name)` would overwrite the fields in the log.